maestro-cli/src/main/java/maestro/cli/insights/TestAnalysisManager.kt

```diff
@@ -0,0 +1,142 @@
+package maestro.cli.insights
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import java.nio.file.Files
+import java.nio.file.Path
+import java.util.stream.Collectors
+import maestro.cli.api.ApiClient
+import maestro.cli.cloud.CloudInteractor
+import maestro.cli.util.EnvUtils
+import maestro.cli.util.PrintUtils
+import maestro.cli.view.box
+import kotlin.io.path.createDirectories
+import kotlin.io.path.exists
+import kotlin.io.path.readText
+import kotlin.io.path.writeText
+
+data class FlowFiles(
+    val imageFiles: List<Pair<ByteArray, Path>>,
+    val textFiles: List<Pair<ByteArray, Path>>
+)
+
+class TestAnalysisManager(private val apiUrl: String, private val apiKey: String?) {
+    private val apiclient by lazy {
+        ApiClient(apiUrl)
+    }
+
+    fun runAnalysis(debugOutputPath: Path): Int {
+        val flowFiles = processFilesByFlowName(debugOutputPath)
```

Comment:
Is this flow file or debug files?

Reply:
debug, good point I'll rename it to be more consistent